posix-mapper/README.md

```diff
@@ -0,0 +1,89 @@
+# POSIX UID and GID Name Mapper (Name TBD)
+
+This is a mapping service that supports GET, and POST operations.  The GET operations will obtain a mapping
+for a given UID or GID, and a POST will add a new entry.
+
+### deployment
+The `posix` war file can be renamed at deployment time in order to support an alternate service name, including introducing 
+additional path elements (see war-rename.conf).
+
+### configuration
+The following runtime configuration must be made available via the `/config` directory.
+
+### catalina.properties
+This file contains java system properties to configure the tomcat server and some of the java libraries used in the service.
+
+See <a href="https://github.com/opencadc/docker-base/tree/master/cadc-tomcat">cadc-tomcat</a>
+for system properties related to the deployment environment.
+
+See <a href="https://github.com/opencadc/core/tree/master/cadc-util">cadc-util</a>
+for common system properties.
+
+`posix` includes multiple IdentityManager implementations to support authenticated access:
+- See <a href="https://github.com/opencadc/ac/tree/master/cadc-access-control-identity">cadc-access-control-identity</a> for CADC access-control system support.
+- See <a href="https://github.com/opencadc/ac/tree/master/cadc-gms">cadc-gms</a> for OIDC token support.
+
+`posix` requires a connection pool to the local user mapping database:
+```
+# database connection pools
+org.opencadc.science-platform.posix.mapping.maxActive={max connections for mapping pool}
+org.opencadc.science-platform.posix.mapping.username={username for mapping pool}
+org.opencadc.science-platform.posix.mapping.password={password for mapping pool}
+org.opencadc.science-platform.posix.mapping.url=jdbc:postgresql://{server}/{database}
+```
+The `mapping` account owns and manages (create, alter, drop) database objects and manages
+all the content (insert, update, delete). The database is specified in the JDBC URL and the schema name is specified 
+in the posix.properties (below). Failure to connect or initialize the database will show up in logs and in the 
+VOSI-availability output.
+
+### cadc-registry.properties
+See <a href="https://github.com/opencadc/reg/tree/master/cadc-registry">cadc-registry</a>.
+
+### posix.properties
```

Comment:
main config file name should be the same as the image, so nominally `posix-mapper.proeprties` unless we come up with a funny proper name.

the config keys should be scoped to this service/code, so it could be `org.opencadc.posix.resourceID` (eg) and also above for connection pool. 

Not sure but I'd probably have made an additional subpackage, maybe `org.opencadc.posix.mapper` (code and config keys).

Reply:
Done.